mne/viz/utils.py

```diff
@@ -1926,6 +1926,28 @@ def _annotation_radio_clicked(label, radio, selector):
     selector.rectprops.update(dict(facecolor=color))
 
 
+def _connection_line(x, fig, sourceax, targetax, y=None):
+    """Connect source and target plots with a line.
+
+    Connect source and target plots with a line, such as time series
+    (source) and topolots (target).
+    """
+    from matplotlib.lines import Line2D
+    transFigure = fig.transFigure.inverted()
+    tf = fig.transFigure
```

Comment:
maybe some more explicit variable name would be better. It's funny that one is transFigure and one is tf.